meta-freescale-3rdparty-mbl/conf/machine/imx6ul-pico-mbl.conf

```diff
@@ -122,13 +121,12 @@ MACHINE_EXTRA_RRECOMMENDS_append = " \
 ##############################################################################
 # Partition layout config
 ##############################################################################
-MBL_WKS_BOOTLOADER1_FILENAME = "SPL"
+MBL_WKS_BOOTLOADER1_FILENAME = "bl2.bin.imx"
 MBL_WKS_BOOTLOADER1_OFFSET_BANK1_KiB = "1"
-MBL_WKS_BOOTLOADER1_SIZE_KiB = "68"
+MBL_WKS_BOOTLOADER1_SIZE_KiB = "1023"
 
-MBL_WKS_BOOTLOADER2_FILENAME = "u-boot.img"
-MBL_WKS_BOOTLOADER2_ALIGN_KiB = "1"
-MBL_WKS_BOOTLOADER2_OFFSET_BANK1_KiB = "69"
+MBL_WKS_BOOTLOADER2_OFFSET_BANK1_KiB = "1024"
+MBL_WKS_BOOTLOADER2_ALIGN_KiB = "1024"
```

Comment:
I suppose we need a similar change as in the commit f749a2264aedcbaf1dabe39679184f489dd49ec1.
@jonathanhaigh-arm, can you please confirm?